bukuserver/translations/babel.cfg

```diff
@@ -0,0 +1,2 @@
+[python: **.py]
+[jinja2: **/templates/**.html]
```

Comment:
When generating translations template file, calls to `gettext()`/`pgettext()`/`lazy_gettext()`/`_()`/`_p()`/`_l()`/`_lp()` are collected from Python and Jinja sources